### DISCUSSION TITLE
Is it possible to build TESS with known volumes

### DISCUSSION THREAD

**Discussion by changhaoliu-UOM** [neperfepx/neper#921] [Q&A] [answered]
Is it possible to build TESS with known volumes? The  known volumes is in a txt and I want to make a tess in a model. The volumes of grains in model should be the numbers in txt.

**Comment by rquey** [comment, marked as answer]
Yes. Use `-morpho size:file(<file_name>)"`. See [-morpho](https://neper.info/doc/neper_t.html#cmdoption-morpho).

**Comment by changhaoliu-UOM** [comment]
Thank you very much.

It works and show a powerful function.

Romain Quey ***@***.***> 于2024年10月14日周一 19:21写道：
